test/extensions/quic_listeners/quiche/envoy_quic_proof_verifier_test.cc

```diff
@@ -166,6 +194,43 @@ TEST_F(EnvoyQuicProofVerifierTest, VerifyProofFailureInvalidLeafCert) {
   EXPECT_EQ("Invalid leaf cert.", error_details);
 }
 
+TEST_F(EnvoyQuicProofVerifierTest, VerifyProofFailureUnsupportedRsaKey) {
```

Comment:
nit: the test uses an unsupported EC curve, but the name says unsupported RSA key. Please fix the test name.

Reply:
done